notebooks/train_notebook.ipynb

```diff
@@ -0,0 +1,1144 @@
+            "Files already downloaded and verified\n",
+            "Files already downloaded and verified\n",
+            "Files already downloaded and verified\n"
+          ]
+        }
+      ],
+      "source": [
+        "from src.data import get_transformed_data, make_data_loaders\n",
+        "from src.transforms import make_transforms\n",
+        "\n",
+        "BATCH_SIZE = 256\n",
+        "EPOCHS  = 100\n",
+        "\n",
+        "valid_ratio = 0.1\n",
+        "\n",
+        "train_data, valid_data, test_data = (\n",
+        "    get_transformed_data(\n",
+        "        make_transforms = make_transforms,\n",
+        "        valid_ratio = valid_ratio\n",
+        "    )\n",
+        ")\n",
+        "\n",
+        "train_iterator, valid_iterator, test_iterator = (\n",
+        "    make_data_loaders(\n",
+        "        train_data,\n",
+        "        valid_data,\n",
+        "        test_data,\n",
+        "        batch_size=BATCH_SIZE,\n",
+        "    )\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "lfPJNbhPGwrW"
+      },
+      "source": [
+        "**Define our Model**\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 7,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "JsPprgn5GwrW",
+        "outputId": "3303861d-2a4a-4df6-d556-7874b6d1ccec"
+      },
+      "outputs": [
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "/home/alodie/miniconda3/envs/avalanche-dev-env/lib/python3.8/site-packages/torch/nn/modules/lazy.py:178: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.\n",
+            "  warnings.warn('Lazy modules are a new feature under heavy development '\n"
+          ]
+        }
+      ],
+      "source": [
+        "from src.model import ResNet, StemConfig, ResidualBlockType\n",
+        "from src.utils import initialize_parameters, epoch_time\n",
+        "\n",
+        "DROPOUT = 0.1\n",
+        "model_architecture = (\n",
+        "    (ResidualBlockType.BASIC, 3, 64, DROPOUT),\n",
+        "    (ResidualBlockType.BASIC, 5, 128, DROPOUT),\n",
+        "    (ResidualBlockType.BOTTLENECK, 26, 256, DROPOUT),\n",
+        "    (ResidualBlockType.BOTTLENECK, 5, 512, DROPOUT),\n",
+        ")\n",
+        "\n",
+        "stem_config = StemConfig(num_channels=64, kernel_size=5, stride=1, padding=2)\n",
+        "model = ResNet(model_architecture, stem_config=stem_config, output_size=10)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 8,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "from pathlib import Path\n",
+        "\n",
+        "run = \"nish_49m_deep6_cyclic_lr_bottleneck_v2.pt\"\n",
+        "path = \"../model/\"\n",
```

Comment:
I'm guessing that because the `model` dir does not exist on my machine, my `torch.save(model.state_dict(), file_path)` returned 
- FileNotFoundError:
- [Errno 2] No such file or directory: '../model/nish_49m_deep6_cyclic_lr_bottleneck_v2.pt'

Reply:
It might be the /model directory that is the issue.